src/test/java/be/syntra/devshop/DevshopBack/services/CartServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package be.syntra.devshop.DevshopBack.services;
+
+import be.syntra.devshop.DevshopBack.entities.User;
+import be.syntra.devshop.DevshopBack.models.CartDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static be.syntra.devshop.DevshopBack.testutilities.CartUtils.createCartDto;
+import static be.syntra.devshop.DevshopBack.testutilities.UserUtils.createUser;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class CartServiceTest {
+
+    @Mock
+    private UserServiceImpl userService;
+
+
```

Comment:
please mind the spacing. One will do between fields

Reply:
fixed